salt/master.sls

```diff
@@ -51,7 +51,9 @@ salt-master:
     - source: salt://{{ tplroot }}/files/master.d
     {%- endif %}
     - clean: {{ salt_settings.clean_config_d_dir }}
-    - exclude_pat: _*
+    - exclude_pat:
+      - _*
```

Comment:
@pprkut could you try wrapping this in single quotes?  The state is failing so must be some list problem.